libpod/options.go

```diff
@@ -469,10 +469,8 @@ func WithNetNS(portMappings []ocicni.PortMapping) CtrCreateOption {
 		if ctr.valid {
 			return ErrCtrFinalized
 		}
-
 		ctr.config.CreateNetNS = true
-		copy(ctr.config.PortMappings, portMappings)
-
+		ctr.config.PortMappings = portMappings
```

Comment:
Use deepcopier here instead of assigning

Reply:
per our discussion, that doesnt appear to work.

Reply:
Agree, am fine with leaving this as = here